scripts/e2e/cmd/runner/helper.go

```diff
@@ -66,14 +68,49 @@ func getClient() (*kubernetes.Clientset, error) {
 	return clientset, nil
 }
 
-func getRoleClient() (*a.RoleAssignmentsClient, error) {
+func getApplicationGatewaysClient() (*n.ApplicationGatewaysClient, error) {
+	env := GetEnv()
+
+	settings, err := auth.GetSettingsFromEnvironment()
+	if err != nil {
+		return nil, err
+	}
+
+	client := n.NewApplicationGatewaysClientWithBaseURI(settings.Environment.ResourceManagerEndpoint, GetEnv().SubscriptionID)
+	var authorizer autorest.Authorizer
+	if env.AzureAuthLocation != "" {
+		authorizer, err = auth.NewAuthorizerFromFile(n.DefaultBaseURI)
```

Comment:
what is the file to get the authorizer, can you help add a comment?

Reply:
add a link to MS docs that shows how to use setup Azure SDK auth using a file.
This is similar to what we do in AGIC when using a service principal for auth.